ironfish/src/rpc/routes/chain/getChainInfo.ts

```diff
@@ -35,6 +36,7 @@ export const GetChainInfoResponseSchema: yup.ObjectSchema<GetChainInfoResponse>
       .object({ index: yup.string().defined(), hash: yup.string().defined() })
       .defined(),
     currentBlockTimestamp: yup.number().defined(),
+    chainDBSizeBytes: yup.number().defined(),
```

Comment:
I think this should go on `node/getStatus` since our other CPU/memory-related metrics are stored there now.

Reply:
Sure, under blockchain section right?